packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -412,10 +412,15 @@ export async function createMember(parent: api.IZoweDatasetTreeNode, datasetProv
     if (name) {
         const label = parent.label as string;
         const profile = parent.getProfile();
+        let replace: shouldReplace;
+        const dsMemberName = `${label}(${name})`;
         try {
-            await ZoweExplorerApiRegister.getMvsApi(profile).createDataSetMember(label + "(" + name + ")", {
-                responseTimeout: profile.profile?.responseTimeout,
-            });
+            replace = await determineReplacement(profile, dsMemberName, "mem");
```

Comment:
If I have previously opened the member and it was downloaded to the temp dir, then "Create New Member" and select "Replace", the contents are not overwritten. Is this expected behavior?

Reply:
Resolved in 955da02